js/web/_i18n/it.json

```diff
@@ -233,13 +233,13 @@
 		"Boxes.Technologies.NoTechs": "Hai raggiunto la fine di questa era",
 		"Boxes.Technologies.Resource": "Risorsa",
 		"Boxes.Technologies.Title": "Costo della ricerca per",
-		"Boxes.Units.AlcaHarvest": "Your Alcatraz is ready to harvest!", //Todo: Translate
+		"Boxes.Units.AlcaHarvest": "Il tuo Alcatraz è pronto per essere raccolto!", 
 		"Boxes.Units.Attack": "Attacco",
 		"Boxes.Units.Bind": "Caserma",
 		"Boxes.Units.Defend": "Difesa",
-		"Boxes.Units.NextUnitsIn": "Le prossime unità __count__ arriveranno in <span class = \"alca-countdown \"> </span> alle ore __harvest__",
+		"Boxes.Units.NextUnitsIn": "Le prossime unità __count__ arriveranno in <span class = \"alca-conto alla rovescia \"> </span> alle ore __raccogliere__",
```

Comment:
Still translated word with __ 
At the end of the line is one

Reply:
Also dont translate the HTML classes (alca-countdown).